### PR TITLE
Update description

### PR DESCRIPTION
**Pull request by khamlyuk**

### CODE REVIEW

File: Readme.md [hunk 1]
```diff
@@ -1,35 +1,35 @@
 <!-- default badges list -->
-![](https://img.shields.io/endpoint?url=https://codecentral.devexpress.com/api/v1/VersionRange/227836631/21.2.3%2B)
 [![](https://img.shields.io/badge/Open_in_DevExpress_Support_Center-FF7200?style=flat-square&logo=DevExpress&logoColor=white)](https://supportcenter.devexpress.com/ticket/details/T845557)
 [![](https://img.shields.io/badge/📖_How_to_use_DevExpress_Examples-e9f6fc?style=flat-square)](https://docs.devexpress.com/GeneralInformation/403183)
 <!-- default badges end -->
 # How to implement a Theme Switcher in Blazor applications
 
-This example demonstrates how to create a Theme Switcher as in [DevExpress Blazor Demos](https://demos.devexpress.com/blazor/) and apply the selected theme to an application dynamically. The example contains solutions both for Blazor Server and Blazor WebAssembly. Note that these solutions target .NET 5.0.
+This example demonstrates how to create a Theme Switcher as in [DevExpress Blazor Demos](https://demos.devexpress.com/blazor/) and apply the selected theme to an application dynamically. The example contains solutions both for Blazor Server and Blazor WebAssembly. Note that these solutions target .NET 6.0.
```
Comment (dirk-pieterse) [inline Readme.md]: `to an application dynamically` Explain

File: Readme.md [hunk 2]
```diff
@@ -1,35 +1,35 @@
...
 
-This Theme Switcher offers the standard Bootstrap theme, [3 DevExpress Bootstrap themes](https://github.com/DevExpress/bootstrap-themes), and [21 Bootstwatch themes](https://bootswatch.com/). 
+![Blazor - Theme Switcher](images/blazor-theme-switcher.png)
 
-Refer to the [Themes](https://docs.devexpress.com/Blazor/401523/common-concepts/themes#implement-a-theme-switcher) documentation topic for more information.
+This Theme Switcher offers the standard Bootstrap theme, [4 DevExpress Bootstrap themes](https://github.com/DevExpress/bootstrap-themes), and [21 Bootstwatch themes](https://bootswatch.com/).
```
Comment (dirk-pieterse) [inline Readme.md]:
```suggestion
This Theme Switcher includes the standard Bootstrap theme, [4 DevExpress Bootstrap themes](https://github.com/DevExpress/bootstrap-themes), and [21 Bootstwatch themes](https://bootswatch.com/). 
```

**Review by dirk-pieterse — APPROVED** [review]

File: Readme.md (hunk 1 above)
Comment (khamlyuk) [reply Readme.md]: Meaning not set it in the code at intialization,
Can we rephrase as follows?
```suggestion
This example demonstrates how to create a Theme Switcher as in [DevExpress Blazor Demos](https://demos.devexpress.com/blazor/) and apply the selected theme to an application at runtime. The example contains solutions both for Blazor Server and Blazor WebAssembly. Note that these solutions target .NET 6.0.
```

Comment (dirk-pieterse) [reply Readme.md]: Ok

**Review by dirk-pieterse — APPROVED** [review]

**Pull request merged**